Content.Server/Kitchen/EntitySystems/MicrowaveSystem.cs

```diff
@@ -384,6 +394,32 @@ public static bool HasContents(MicrowaveComponent component)
             return component.Storage.ContainedEntities.Any();
         }
 
+        /// <summary>
+        /// Handles the attempted cooking of unsafe objects
+        /// </summary>
+        /// <remarks>
+        /// Returns true if the microwave can continue to cook, false if it can't.
+        /// </remarks>
+        private bool HandleUnsafeItems(EntityUid uid, MicrowaveComponent component)
```

Comment:
Since this returns a `bool`, renaming this `RollExplode` or `ShouldExplode` and flipping the boolean makes it more clear what this is doing.